leetcode/string.md

```diff
@@ -0,0 +1,46 @@
+# String
```

Comment:
Your Pull Request title: Added solution for valid-palindrome

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']